services-camel/src/main/resources/log4j2.properties

```diff
@@ -1,7 +1,7 @@
 
 appender.file.type = File
 appender.file.name = file
```

Comment:
I think you might need to commend out all the stuff that begins with "appender.file"